packages/docs.siwt.xyz/docs/access-control-query.md

```diff
@@ -24,17 +24,23 @@ Setting your requirements is done by defining a query in the following format:
 
 ```
 const result = await queryAccessControl({
-  network: Network.mainnet,             // Choose which network should be checked. Options are Mainnet and Ghostnet. Defaults to Ghostnet.
-  parameters: {
-    pkh: 'WALLET ADDRESS',              // Variables that are required for the query. Currently this is limited to the user's wallet address. Required.
+  query: {
```

Comment:
Just a question here: There must be somewhere the documentation about the token validity that we introduced in v0.03 - should we make a note at that point that the new POC of ODRL policies for fine grained access should be looked at and link the approach there?